docs-core/src/main/java/com/sismics/docs/core/model/jpa/Review.java

```diff
@@ -0,0 +1,162 @@
+package com.sismics.docs.core.model.jpa;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import com.google.common.base.MoreObjects;
```

Comment:
TODO:
- Add file `dbupdate-028-0.sql` in the folder `docs-core/src/main/resources/db/update` with the SQL code to create the SQL table
- Update all three `config.properties` files to say `db.version=28`